### ISSUE TITLE
The requirements.txt is not valid, Use this file as requirements.txt

### ISSUE THREAD

**Issue by Dilshad737** [Asutosh11/face-match-x#16]
> absl-py==1.4.0
astunparse==1.6.3
cachetools==5.3.0
certifi==2022.12.7
charset-normalizer==3.0.1
click==8.1.3
colorama==0.4.6
contourpy==1.0.7
cycler==0.11.0
Flask==2.2.2
flatbuffers==23.1.21
fonttools==4.38.0
gast==0.4.0
google-auth==2.16.0
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
grpcio==1.51.1
h5py==3.8.0
idna==3.4
itsdangerous==2.1.2
Jinja2==3.1.2
keras==2.11.0
Keras-Applications==1.0.8
keras-vggface==0.6
kiwisolver==1.4.4
libclang==15.0.6.1
Markdown==3.4.1
MarkupSafe==2.1.2
matplotlib==3.6.3
mtcnn==0.1.1
numpy==1.24.1
oauthlib==3.2.2
opencv-python==4.7.0.68
opt-einsum==3.3.0
packaging==23.0
Pillow==9.4.0
protobuf==3.19.6
pyasn1==0.4.8
pyasn1-modules==0.2.8
pyparsing==3.0.9
python-dateutil==2.8.2
PyYAML==6.0
requests==2.28.2
requests-oauthlib==1.3.1
rsa==4.9
scipy==1.10.0
six==1.16.0
tensorboard==2.11.2
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.11.0
tensorflow-estimator==2.11.0
tensorflow-intel==2.11.0
tensorflow-io-gcs-filesystem==0.30.0
termcolor==2.2.0
typing_extensions==4.4.0
urllib3==1.26.14
Werkzeug==2.2.2
wrapt==1.14.1

Also you need to make another change to run it successfully:
`from keras.utils.layer_utils import get_source_inputs`
IN place of:
`from keras.engine.topology import get_source_inputs`

in `Lib\site-packages\keras_vggface\models.py`

**Comment by Asutosh11** [comment]
THANKS for the update @Dilshad737 .. I haven't opened this project from some time now. You can raise a PR if you want, we will merge it.
